_redirects

```diff
@@ -223,6 +223,20 @@
 /en/programs/okta-personal/Content/Topics/okta-personal/create-account.htm https://support.oktapersonal.com/article/24-create-account
 /en/programs/okta-personal/Content/Topics/okta-personal/get-started.htm https://support.oktapersonal.com/category/15-getting-started
 
+/en/programs/em/content/topics/identity-governance/em/entitlement-mgt.htm /oie/en-us/content/topics/identity-governance/em/entitlement-mgt.htm
```

Comment:
Do all of redirects have the same base path? If so can we write one rule similar to https://github.com/okta/okta-help/blob/3cb6cb7df456610891dd3452f4853c0228e8c9c3/_redirects#L54

```
/en/programs/em/content/topics/identity-governance/em/* /oie/en-us/content/topics/identity-governance/em/:splat
```

Reply:
Also if we still have content in /en/programs/ redirects will not work. That's better for end user but harder to test for us